docs/gettingstarted/structuring.md

```diff
@@ -47,18 +47,20 @@ The `mods.toml` file is formatted as [TOML](https://github.com/toml-lang/toml),
       [[dependencies.examplemod]]
         modId="forge"
         mandatory=true
-        versionRange="[31,)"
+        versionRange="[34,)"
         ordering="NONE"
         side="BOTH"
 
       [[dependencies.examplemod]]
         modId="minecraft"
         mandatory=true
-        versionRange="[1.15.2]"
+        versionRange="[1.16.3,]"
```

Comment:
Shouldn't this be like cpw made the mdk, and end in `,1.17)`

Reply:
In his tweet he specifically mentioned having the trailing slash, so I don't know why the mdk has it differently. The result is the same right?